frontend/pages/hosts/HostDetailsPage/SoftwareListRow/SoftwareListRow.jsx

```diff
@@ -11,7 +12,7 @@ class SoftwareListRow extends Component {
 
   render() {
     const { software } = this.props;
-    const { name, source, version } = software;
+    const { name, source, version, vulnerabilities } = software;
```

Comment:
`vulnerabilities` should be added to the interface of software so we can know what type to expect.